payment-instruments/mobile-pay/seamless-view.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Seamless
+redirect_from: /payments/mobile-pay/seamless-view
+estimated_read: 10
+description: |
+  **Seamless View** scenario gives your customers the opportunity to pay with
+  MobilePay directly from within your webshop. In the Seamless view scenario,
+  Swedbank Pay receives a mobile number (MSISDN) from the payer through
+  Swedbank Pay Payments. Swedbank Pay performs a payment that the payer then
+  must confirm through the MobilePay mobile app.
```

Comment:
Note: Was unsure on how to properly word this section, so it is based of Vipps. The major difference is that Prefill information is provided by the merchant, and not the payer in the UI, as they now enter the information into MobilePay's own UI (after clicking the "pay" button), but it can be filled out for the user if the merchant includes MSISDN in the create request.